.changelog/852.txt

```diff
@@ -1,3 +1,5 @@
 ```release-note:feature
  New resource: Add `hcp_packer_bucket` resource for managing HCP Packer buckets
+ New Resource: Add `hcp_packer_bucket_iam_policy` resource for assigning a list of policy bindings to multiple principals for a HCP Packer Bucket
```

Comment:
Since PR 852 is the one that will get merged into pain, I figured it made sense to update the changelog here